### ISSUE TITLE
Android tablet not respecting CH settings

### ISSUE THREAD

**Issue by teknetia** [maykar/custom-header#93]
I have run into an issue where my Android wall display/tablet is not applying the CH configuration, even after a full clear of the browser cache. I am using Fully Kiosk Browser to load the HASS UI. It previously worked and I am a little unclear on what may have changed to cause an issue.

As far as I can see there aren't any errors in the logs and if I log in as the same user (wall-display) in Chrome/Brave/Safari it correctly loads the UI mods I expect. The issue also happens in the stock browser (Lightning).

## Info
Home Assistant 0.103.6 (via Hass.io)
Custom Header 1.1.7 (via HACS)
Fully Kiosk Browser 1.3.4

## Configuration
```
custom_header:
  chevrons: false
  exceptions:
    - conditions:
        user: 'Wall Display, wall-display'
      config:
        compact_mode: true
        disable_sidebar: true
        hide_config: true
        hide_help: true
        hide_raw: true
        hide_unused: true
        menu_hide: true
        voice_hide: true
        options_hide: true
  hidden_tab_redirect: false
  shadow: false
resources:
  - type: module
    url: /community_plugin/custom-header/custom-header.js
```

**Comment by maykar** [comment]
Please, try the latest release 1.2.0 and report back.

**Comment by teknetia** [comment]
No luck. I’m not 100% sure when it broke but it was possibly when I upgraded to 1.1.7 in HACS. I don’t know how much older the working version was though for the sake of testing. 

Anything useful I can pull from it?

**Comment by teknetia** [comment]
So I did some further playing around with things and I have it working again. I am not sure exactly which of the following steps worked but I have a feeling it might have been the user level.

- Uninstalled Card Tools & Theme Maker as I don't need them anymore, both are custom install, and I wasn't convinced they weren't part of the issue. I didn't notice any issue with Custom Header but I did notice once they were removed a strange duplication (some header elements appeared below the layouts) did stop.

- Downgraded CH to 1.1.5 and reconfigured from scratch. No change so I uninstalled it completely and then reinstalled 1.2.0 and reconfigured again.

- Changed the Wall Display user from an Admin to a User. I thought this was the most likely thing but if I change it back to an admin the problem doesn't reappear.

So something in those steps got it back to displaying correctly again!
